platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/Mapbox.java

```diff
@@ -145,23 +135,12 @@ public static synchronized Boolean isConnected() {
     return (activeNetwork != null && activeNetwork.isConnected());
   }
 
-  /**
-   * Returns a location source instance with empty methods.
-   *
-   * @return an empty location source implementation
-   * @deprecated Replaced by {@link Mapbox#getLocationEngine()}
-   */
-  @Deprecated
-  public static LocationSource getLocationSource() {
-    return new EmptyLocationSource();
-  }
-
-
   /**
    * Returns the location engine used by the SDK.
    *
    * @return the location engine configured
    */
+  // TODO Do we need to expose this?
```

Comment:
Remove or ticket.

Reply:
Ticketed

Reply:
@electrostat 
> Ticketed

This should be ticketed in this repo 👉 https://github.com/mapbox/mapbox-gl-native/issues